paasta_tools/cli/cmds/local_run.py

```diff
@@ -381,6 +381,37 @@ def get_container_name():
     return 'paasta_local_run_%s_%s' % (get_username(), randint(1, 999999))
 
 
+def check_for_sh(command):
```

Comment:
Is there a reason you have to even check for this? Why can't we just quote whatever they provide and still always pass it in to /bin/sh (i.e. `sh -c "sh -c 'echo foo'"`)?